bin/mk-tags.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+HSCOPE_DIR=$HOME/.hscope
+CODEX_CONFIG=$HOME/.codex
+
+if test -e $CODEX_CONFIG
+then
+  echo "Please move or remove $CODEX_CONFIG and then run again"
+  exit 1
+fi
+
+# XXX it does not seem to include the current project anyway
+cat << EOF > $CODEX_CONFIG
+currentProjectIncluded: true
+hackagePath: $HSCOPE_DIR/packages/hackage.haskell.org/
+tagsFileHeader: true
+tagsFileName: tags
+tagsFileSorted: true
+tagsCmd: hasktags --ctags --follow-symlinks --output="\$TAGS" "\$SOURCES"
```

Comment:
Also, add `--etags`?

Reply:
You can add emacs option to it. codex has an option to set emacs format, passing that option we can generate that config file and paste it in this script.